doc/qbk/02_2_strings.qbk

```diff
@@ -12,22 +13,58 @@
 [section:strings Using Strings]
 
 Modifiable sequences of characters are represented using instances of type
-[link json.ref.boost__json__string `string`].
-These strings have an interface almost identical to `std::string`, and
-operate the same way, with these variations:
+__string__.
+These strings have an modified `string_view` based version of the `std::string` interface, and
+operate the same way, except that:
 
-* __string__ is not a class template.
+* __string__ is not a class template,
+
+* __string__ only uses `char` as its character type,
+
+* redundant overloads for string operations have been replaced with a `string_view` based interface,
+
+* access to characters in the range `[size(), capacity()]` is permitted, and
 
 * __storage__ is used instead of __Allocator__.
 
-* New characters are 
+With the `string_view` based interface, operations requiring an input string are implemented as a single overload with
+a parameter of type `string_view`, and can accept most string-like objects in addition to other `string_view`s. 
+Objects such as null terminated character pointers, `std::string`, `json::string`, subranges of strings, and objects
+convertible to `string_view` can all be passed to these functions. 
 
-A string may be constructed initially empty without incurrening any
-memory allocations using the default storage, or a specified storage:
+[snippet_strings_4]
 
-[snippet_strings_1]
+More formally, `std::string` member function overloads that accept any of the
+following parameter combinations as an input string:
+
+* a `std::string` parameter, or
+
+* a `std::string` parameter and two `size_type` parameters that specify a substring within that string, or
+
+* a parameter of a type convertible to `string_view`, or
+
+* a parameter of a type convertible to `string_view` and two `size_type` parameters that specify a substring within the `string_view`, or
+
+* a `const_pointer`, or
 
-As with `std` strings, space may be reserved upon construction.
-Exceptions are possible here:
+* a parameter of type `const_pointer` and a `size_type` parameter that specifies the length of the string
+
+are replaced with with an overload accepting a `string_view` parameter.
+
+This design removes several redundant overloads from the interface. For example, the 11 overloads of `std::string::insert` are reduced
+to just 3 in __string__, while still providing identical functionality. In addition to these changes, overloads taking a `std::initializer_list<char>` parameter have been removed. Such overloads have little use,
+as they serve as little more than a wrappers for arrays, with an inefficient syntax:
+
+[snippet_strings_3]
+
+With the removal of overloads that specify parameters for a substring, a member function `subview` that returns a `string_view` referring to
+a substring within the __string__ is provided to facilitate cheap substring operations:
```

Comment:
string...within...string...to..substring the string view thing string viewed string string TOO MANY STRINGS!! ARGH

Reply:
You're right, it's a little excess. I trimmed that line down.